AdSecGHTests/Helpers/DropIntoGhDocumentTest.cs

```diff
@@ -71,6 +71,7 @@ public void ShouldHaveNoWarnings() {
 
     [Fact]
     public void ShouldHaveNoErrors() {
+      Assert.NotNull(_document);
       TestNoRuntimeMessagesInDocument(_document, GH_RuntimeMessageLevel.Error);
```

Comment:
refactor `TestNoRuntimeMessagesInDocument` to return a bool and then assert against that boolean. Remove the _document no null.